tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -1,43 +0,0 @@
-<?php
-namespace Aws\Symfony\DependencyInjection;
-
-use AppKernel;
-use PHPUnit\Framework\TestCase;
-use Symfony\Component\Filesystem\Filesystem;
-
-class ConfigurationTest extends TestCase
-{
-    public function setUp()
-    {
-        (new Filesystem)
-            ->remove(implode(DIRECTORY_SEPARATOR, [
-                dirname(__DIR__),
-                'fixtures',
-                'cache',
-                'test',
-            ]));
-    }
-
-    /**
-     * @test
-     * @dataProvider formatProvider
-     *
-     * @param string $format
-     */
-    public function container_should_compile_and_load($format)
-    {
-        $kernel = new AppKernel('test', true, $format);
-        $kernel->boot();
-
-        $this->assertTrue($kernel->getContainer()->has('aws_sdk'));
```

Comment:
Why not just modify this line to make a more idiomatic assertion? The sample configs could declare a public service into which the SDK is injected, and this test could fetch that service and make an assertion about its state.